sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_options.hpp

```diff
@@ -18,62 +18,24 @@
 
 namespace Azure { namespace Security { namespace KeyVault { namespace Certificates {
 
-  /**
-   * @brief The API version to use from Key Vault.
-   *
-   */
-  class ServiceVersion final {
-  private:
-    std::string m_version;
-
-  public:
-    /**
-     * @brief Construct a new Service Version object
-     *
-     * @param version The string version for the Key Vault Certificate service.
-     */
-    ServiceVersion(std::string version) : m_version(std::move(version)) {}
-
-    /**
-     * @brief Enable comparing the ext enum.
-     *
-     * @param other Another #ServiceVersion to be compared.
-     */
-    bool operator==(ServiceVersion const& other) const { return m_version == other.m_version; }
-
-    /**
-     * @brief Return the #ServiceVersion string representation.
-     *
-     */
-    std::string const& ToString() const { return m_version; }
-
-    /**
-     * @brief Use to send request to the 7.2 version of Key Vault service.
-     *
-     */
-    AZ_SECURITY_KEYVAULT_CERTIFICATES_DLLEXPORT static const ServiceVersion V7_2;
-
-    /**
-     * @brief Use to send request to the 7.3 version of Key Vault service.
-     *
-     */
-    AZ_SECURITY_KEYVAULT_CERTIFICATES_DLLEXPORT static const ServiceVersion V7_3;
-  };
-
   /**
    * @brief Define the options to create an SDK Certificate client.
    *
    */
   struct CertificateClientOptions final : public Azure::Core::_internal::ClientOptions
   {
-    ServiceVersion Version;
+    /**
+     * @brief Service Version used.
+     *
+     */
+    const std::string Version;
 
     /**
      * @brief Construct a new Certificate Client Options object.
      *
      * @param version Optional version for the client.
      */
-    CertificateClientOptions(ServiceVersion version = ServiceVersion::V7_3)
+    CertificateClientOptions(std::string version = "7.3")
```

Comment:
Why is this a ctor param? Can't we just set this field in the ctor and let the customer overwrite the field if they choose? Just treat it ike any othe option

Reply:
true, done